README.md

```diff
@@ -1,19 +1,19 @@
-# FLink Table Store
+# Flink Table Store
 
 Flink Table Store is a unified streaming and batch store for building dynamic tables on Apache Flink.
 
 Flink Table Store is developed under the umbrella of [Apache Flink](https://flink.apache.org/).
 
-## <a name="build"></a>Building the Project
```

Comment:
It is better to leave this, the title can be `build`.

Reply:
In fact you do not need this, the `Building the Project` is not to long, or you can use `Build` as the title for unification with other tiles.

Reply:
I make it shorter, sir

Reply:
shorter is good (: